python/src/scipp/compat/mantid.py

```diff
@@ -164,14 +171,35 @@ def init_pos(ws):
                        dtype=sc.dtype.vector_3_float64)
 
 
+def _get_dtype_from_values(values):
+    if hasattr(values, 'dtype'):
+        dtype = values.dtype
+    else:
+        if len(values) > 0:
+            dtype = type(values[0])
+            if dtype is str:
+                dtype = sc.dtype.string
+            elif dtype is int:
+                dtype = sc.dtype.int32
```

Comment:
So this is now always forcing to `int32`? Previously this was done only for spectrum numbers. Is this the correct choice?

Reply:
I am not sure. Spectrum numbers will now go through the `if hasattr(values, 'dtype')` because they are numpy arrays. 

Maybe I can just make it int64 to be on the safe side? I'm not sure if the int and float cases are ever used. They would have to be manual labels with integers/doubles that are Python lists but not numpy arrays.

Reply:
I think on Windows Python's `int` is 32 bit?

Reply:
But maybe it is best to force to 64 bit in that case anyway (see repeated issues with Python unit tests on Windows).